snowfall/decoding/graph.py

```diff
@@ -19,7 +19,7 @@ def compile_LG(L: Fsa, G: Fsa, ctc_topo_inv:Fsa, labels_disambig_id_start: int,
         G:
             An ``Fsa`` that represents the language model (G), i.e. it's an acceptor
             with words as ``symbols``.
-        ctc_topo_inv:  Epsilons are in `aux_labels` and `labels` contain phone IDs.
+        ctc_topo:  Epsilons are in `labels` and `aux_labels` contain phone IDs.
```

Comment:
I'm confused by this line, I was thinking the left side, i.e. labels,  would contain `blank` instead of `epsilon`